torch/autograd/__init__.py

```diff
@@ -147,13 +150,13 @@ def backward(
         tuple(inputs) if inputs is not None else tuple()
 
     grad_tensors_ = _tensor_or_tensors_to_tuple(grad_tensors, len(tensors))
-    grad_tensors_ = _make_grads(tensors, grad_tensors_)
+    grad_tensors_ = _make_grads(tensors, grad_tensors_, batched_grads=False)
     if retain_graph is None:
         retain_graph = create_graph
 
     Variable._execution_engine.run_backward(
         tensors, grad_tensors_, retain_graph, create_graph, inputs,
-        allow_unreachable=True, accumulate_grad=True)  # allow_unreachable flag
```

Comment:
nit if this comment is for the error message, maybe mention that this is going to c++?

Reply:
Ended up going with Vadim's suggestion, but also added a comment to point this out

Reply:
I actually think that the comment is better than the one liner.
Pointing the users to `Variable._execution_engine` is going to lead them nowhere.

Reply:
The comment does seem to be more useful, but I guess for the more experienced user, it is more about aesthetics than usefulness lol.

If we only care about newer versions of python, the multi-line issue correctly displays the first line on Python 3.8 it seems https://bugs.python.org/issue12458.

I'm okay with just doing:
```
  File "/home/local/pytorch/torch/autograd/__init__.py", line 267, in grad
    return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
  File "/home/local/pytorch/torch/autograd/function.py", line 199, in apply
    return user_fn(self, *args)
  File "/home/local/test.py", line 10, in backward
    assert False
AssertionError
```

Reply:
From offline discussion: we should duplicate the comment on both lines, and add comment above